test/platform/test-platform-fixtures.ts

```diff
@@ -54,32 +18,9 @@ export function createIdentifyHandler(
       payload: {
         device: {
           id: deviceId,
-          verificationId: veriicationId,
+          verificationId: verificationId,
```

Comment:
As the key and variable are the same, you can just do `verificationId,` like in other places.

Reply:
Done